### ISSUE TITLE
[Feature Request] Task relations such as 'preceding', 'blocked', 'blocked by'

### ISSUE THREAD

**Issue by HammyHavoc** [nextcloud/tasks#1915]
Would be great to see something like this: https://vikunja.io/docs/task-relation-kinds/

Otherwise, task lists become a hellscape of things a user might not yet be able to do until x is done.

**Comment by Thaodan** [comment]
I think this could be done as part of #2135. The task that I just created could be considering a duplicate all trough mine is more technical.

**Comment by HammyHavoc** [comment]
> I think this could be done as part of #2135. The task that I just created could be considering a duplicate all trough mine is more technical.

Yes, I'd still love to see this. Ended up using MyLifeOrganized but no sharing/Collab. Nextcloud Tasks has a huge amount of untapped potential.

**Comment by PeonyThePeon** [comment]
The best way to implement this feature request would be to support the [RELTYPE=DEPENDS-ON parameter from RFC 9253](https://www.rfc-editor.org/info/rfc9253/#name-additional-new-reltype-para).

I have now filed a feature request specifically for that: https://github.com/nextcloud/tasks/issues/3188 .
